### DISCUSSION TITLE
problems using JIT with NuxtJS & Vite

### DISCUSSION THREAD

**Discussion by ciarantr** [tailwindlabs/tailwindcss#4859] [Help]
I am having issues with setting up tailwind JIT using Vite & NuxtJS. Not sure if I set it tailwind config correctly, if anyone has had this issue and resolved it?

tailwind.js config

  module.exports = {
    mode: "jit",
    purge: [
      "./components/**/*.{vue,js}",
      "./layouts/**/*.vue",
      "./pages/**/*.vue",
      "./plugins/**/*.{js,ts}",
      "./nuxt.config.{js,ts}",
    ],
    darkMode: false, // or 'media' or 'class'
    theme: {
      fontFamily: {
        logo: ["Rock Salt", "cursive"],
      },
      extend: {},
    },
    variants: {
      extend: {},
    },
    plugins: [],
  };

**Comment by simonswiss** [comment]
Hey!

What issue are you having exactly? Are you able to share a GitHub reproduction of the problem?

**Reply by ciarantr** [reply]
Hi, so at the moment I am unable to use tailwind classes in the Nuxt templates using the JIT compiler.

Tailwind works ok when I disable JIT complier in the tailwind config though.

I can add the classes to the HTML, however the classes are not applied when using the JIT compiler. The sonly way I can use tailwind is by using a scoped style to the template. I have attached a GitHub link if that is easier to find out what I may be doing wrong. Any advice would be greatly appreciated!

https://github.com/ciaran-io/webdesign-app
